mc_lib/lattices/meson.build

```diff
@@ -0,0 +1,14 @@
+install_dir = 'mc_lib/lattices'
```

Comment:
Ah this is the issue with `install_dir` in the docs. I would prefer using strings directly rather than use a variable here, less confusing. Since if you forget this line in one file, `install_dir` is still defined elsewhere and will be picked up silently